src/test/java/TestBasicAnnotation.java

```diff
@@ -0,0 +1,104 @@
+
+import classfortest.Person;
+import classfortest.StudentBreskulTeam;
+import org.breskul.connectivity.annotation.Column;
+import org.breskul.connectivity.annotation.Entity;
+import org.breskul.connectivity.annotation.Id;
+import org.breskul.connectivity.annotation.Table;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class TestBasicAnnotation {
+
+    @Test
+    public void existAnnotationTable() {
+
+        final var annotationEntityIsPresent = Person.class.isAnnotationPresent(Entity.class);
+        assert (annotationEntityIsPresent);
+        final var annotationTableIsPresent = Person.class.isAnnotationPresent(Table.class);
+        assert (annotationTableIsPresent);
+        final var tableName = Person.class.getDeclaredAnnotation(Table.class).value();
+        assert (tableName.equals("persons"));
+    }
+
+    @Test
+    public void emptyAnnotationTable() {
+
+        final var annotationEntityIsPresent = StudentBreskulTeam.class.isAnnotationPresent(Entity.class);
+        assert (annotationEntityIsPresent);
```

Comment:
Why don't you use Assertions from junit library? We have already added it to pom.xml